src/Server/Resource/DoctrineResource.php

```diff
@@ -330,7 +332,8 @@ public function create($data)
             return $data;
         }
 
-        $entity = new $entityClass;
+        $entity = (null !== $entityFactory) ? $entityFactory->get($entityClass) : new $entityClass;
```

Comment:
We need a factory because we do want the new instance to be built using the class constructor: many people use it to assign default values to properties on the entities.

Maybe if someone wants to use `doctrine/instantiator` for building this, they can create a factory that uses it?